app/serializers/api/internal/v1/hearing_summary_serializer.rb

```diff
@@ -14,6 +14,12 @@ class HearingSummarySerializer
           end
         end
 
+        attribute :sitting_days do |hearing_summary|
+          hearing_summary.sitting_days.map do |day|
```

Comment:
Does this not want to be `hearing_summary.hearing_days` - hearing summary doesn't have a sitting_days attribute